viz/charts/linkerd-viz/templates/service-profiles.yaml

```diff
@@ -21,52 +22,24 @@ spec:
     condition:
       method: POST
       pathRegex: /api/v1/ListServices
-  - name: POST /api/v1/Version
-    condition:
-      method: POST
-      pathRegex: /api/v1/Version
   - name: POST /api/v1/SelfCheck
     condition:
       method: POST
       pathRegex: /api/v1/SelfCheck
----
-apiVersion: linkerd.io/v1alpha2
-kind: ServiceProfile
-metadata:
-  name: linkerd-dst.linkerd.svc.cluster.local
-  namespace: linkerd
-spec:
-  routes:
-  - name: POST /io.linkerd.proxy.destination.Destination/Get
-    condition:
-      method: POST
-      pathRegex: /io\.linkerd\.proxy\.destination\.Destination/Get
-  - name: POST /io.linkerd.proxy.destination.Destination/GetProfile
-    condition:
-      method: POST
-      pathRegex: /io\.linkerd\.proxy\.destination\.Destination/GetProfile
----
-apiVersion: linkerd.io/v1alpha2
-kind: ServiceProfile
-metadata:
-  name: linkerd-dst-headless.linkerd.svc.cluster.local
-  namespace: linkerd
-spec:
-  routes:
-  - name: POST /io.linkerd.proxy.destination.Destination/Get
+  - name: POST /api/v1/Gateways
     condition:
       method: POST
-      pathRegex: /io\.linkerd\.proxy\.destination\.Destination/Get
-  - name: POST /io.linkerd.proxy.destination.Destination/GetProfile
+      pathRegex: /api/v1/Gateways
+  - name: POST /api/v1/Edges
     condition:
       method: POST
-      pathRegex: /io\.linkerd\.proxy\.destination\.Destination/GetProfile
+      pathRegex: /api/v1/Edges
 ---
 apiVersion: linkerd.io/v1alpha2
 kind: ServiceProfile
 metadata:
-  name: linkerd-prometheus.linkerd.svc.cluster.local
-  namespace: linkerd
+  name: prometheus.{{.Values.namespace}}.svc.{{.Values.clusterDomain}}
+  namespace: {{.Values.namespace}}
```

Comment:
I think the prometheus and grafana parts should be guarded around those components `.enabled` condition

Reply:
Makes sense. Updated!